src/array.jl

```diff
@@ -580,13 +580,6 @@ Return levels which appear in `A`, in the same order as [`levels`](@ref)
 """
 function unique end
 
-"""
```

Comment:
Originally I put the `droplevels!` docstrings before the `droplevels!` function, line 719, rather than before `droplevels`, because it didn't get documented (but I copied rather than moved, so that you could see what I'd done). Then in the edit I removed the original.